src/style.css

```diff
@@ -1,3 +1,109 @@
-body {
+/* General reset */
+* {
   margin: 0;
 }
+
+body {
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+
+  --color-blue: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo__link {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
```

Comment:
1. every BEM block should be in separate css file
2. here you have just 1 element, so I would rather make it inside header block -> `header__logo-link`, so you will have block header and element logo-link